bundles/org.openhab.binding.irtrans/README.md

```diff
@@ -16,30 +21,68 @@ There is no specific binding configuration required.
-| receiver        | String    | Receive (filtered) infrared commands on the receiver LED of the transceiver         |
+### `blaster` Thing Channels
 
-The _blaster_ Channel Type requires a _led_ configuration parameter to specify on which infrared commands will be emitted, _remote_ the remote or manufacturer name which's commands will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be '\*' for 'any' remote), and _command_ the name of the command will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be '*' for 'any' command).
+| Channel Type ID | Item Type | Description                                                                                                               |                                      |
+|-----------------|-----------|---------------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+| io              | String    | Allows to read infrared commands received by the blaster, as well as to write infrared commands to be sent by the blaster | [Channel configuration](#io-channel) |
 
-The _receiver_ Channel Type requires _remote_ the remote or manufacturer name which's commands will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be '\*' for 'any' remote), and _command_ the name of the command will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be '*' for 'any' command).
+#### `io` Channel
 
-The _blaster_ Thing supports a _io_ Channel (of Item Type String) that allows to read infrared commands received by the blaster, as well as to write infrared commands to be sent by the blaster.
+| Parameter | Description                                                                                                                                                                      | Config   | Default |
+|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| led       | Specifies the led on which infrared commands will be emitted                                                                                                                     | Required | -       |
+| remote    | The remote or manufacturer name which's commands will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be ' \*' for 'any' remote) | Required | -       |
+| command   | The name of the command will be allowed, as defined in the IRtrans server database that is flashed into the transceiver (can be '*' for 'any' command)                           | Required | -       |
```

Comment:
I do not find any channel configuration defined for this channel type in XML files. Is it something added dynamically ?

Reply:
Yes thanks, removed. Somehow i got this dupliced to this channel.